roles/container-engine/containerd-common/tasks/main.yml

```diff
@@ -17,3 +29,23 @@
       skip: true
   tags:
     - facts
+
+- name: containerd-common | remove any distribution specific containerd package
+  package:
+    name: "{{ containerd_package }}"
+    state: absent
+  when:
+    - not is_ostree
```

Comment:
Moved this part back to the `containerd` role since `containerd-common` is also referenced by the `docker` role. That change also cleans up the repos.